tests/test_odgovlt.py

```diff
@@ -195,3 +197,48 @@ def test_OdgovltHarvester(app, db, mocker):
     assert result['report_status'] == 'added'
     assert result['errors'] == []
     assert was_last_job_considered_error_free()
+    ckanapi = CkanAPI()
+    ids = ckanapi.package_list()
+    package = [ckanapi.package_show(id=ids[0])]
+    assert [(x['title'],) for x in package] == [
```

Comment:
What is the point of having single package in a list and then writing a list comprehension over that single list item in every assert statement? Wouldn't it be easier to simply do:

```python
assert package['title'] == '2014 m. vidutinio metinio paros eismo intensyvumo duomenys'
```

?